vmdb/app/views/layouts/_perf_charts.html.haml

```diff
@@ -0,0 +1,110 @@
+%div{:id => "#{chart_set}_charts_div", :onmousedown => "if (typeof miqMenu != 'undefined') miqMenu.hideContextMenu();"}
+  - if @temp[:charts] || @perf_options[:chart_type] != :performance
+    - perf_options ||= @perf_options
+    - width = perf_options[:index] ? 1000 : 350
+    - height = perf_options[:index] ? 700 : 250

```

Comment:
`width, height = perf_options[:index] ? [1000, 700] : [350, 250]`
or just normal `if..else` but no need to repeat the condition